deploys/demo-test.syncfor.science/config/defaults/clients.json

```diff
@@ -1,50 +0,0 @@
-[{
```

Comment:
Scaled back fixtures.yml to just create the user. Now clients are registered in a task described in `demo.syncfor.science/README.md` and `demo-test.syncfor.science/README.md`. Tested on demo-test, everything seems to be working.